python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -372,15 +394,42 @@ def _buildCellSet(self, exposure, referencePsfModel):
                 widthList.append(widthS)
                 heightList.append(heightS)
 
-        lenMax = max(max(heightList), max(widthList))
-        lenMin = min(min(heightList), min(widthList))
+        psfSize = max(max(heightList), max(widthList))
 
-        lenPsfScience = lenMax if self.config.padPsf else lenMin
-        dimenS = afwGeom.Extent2I(lenPsfScience, lenPsfScience)
+        if self.config.doAutoPadPsf:
+            minPsfSize = nextOddInteger(self.kConfig.kernelSize*self.config.autoPadPsfTo)
```

Comment:
I acknowledge two suggestions here: (1) to take the minPsfSize calculation out of a function and (2) to switch the last bit to 1 to get the next odd integer. 
1: This function is not in a loop  and only called once.  My rationale for putting it in a function was to reduce code duplication with the unit test. If you argue that unit tests *should* calculate it differently, then I'll take it out of the function. 
2: Flipping the last bit to 1 smells clever to me, because it doesn't read like english. 

The function plus english-reading % 2 version also appears to be (counterintuitively) faster, but a microsecond (or even 100 milliseconds) won't make a difference in a long task like ModelPsfMatch:

```
import timeit

setupBitFlip = """
from random import random
import numpy as np
def nextOddInt(x):
    return int(np.ceil(x))|1

def genKRand():
    for i in range(1000):
        yield 10*random()
"""

setupIfMod = """
from random import random
import numpy as np

def genKRand():
    for i in range(1000):
        yield 10*random()

def nextOddInt(x):
    return x if x % 2 == 0 else x + 1
"""

print('%.02f us'% (1000*timeit.timeit('[nextOddInt(y) for y in genKRand()]', setup=setupIfMod, number=10)/10.))
# 0.63 us
print('%.02f us'% (1000*timeit.timeit('[nextOddInt(y) for y in genKRand()]', setup=setupBitFlip, number=10)/10.))
# 1.26 us
print('%.02f us'% (1000*timeit.timeit('[int(np.ceil(y))|1 for y in genKRand()]', setup=setupBitFlip, number=10)/10.))
# 1.07 us

# Single value
print('%.02f us'%(timeit.timeit('nextOddInt(3.4)', setup=setupIfMod, number=1000000)))
# 0.35 us
print('%.02f us'%(timeit.timeit('nextOddInt(3.4)', setup=setupBitFlip, number=1000000)))
# 0.99 us
print('%.02f us'%(timeit.timeit('int(np.ceil(3.4))|1', setup=setupBitFlip, number=1000000)))
# 0.82 us
```